avaframe/com1DFA/com1DFA.py

```diff
@@ -2578,9 +2561,17 @@ def prepareVarSimDict(standardCfg, inputSimFiles, variationDict, simNameExisting
             cfgSim["INPUT"].pop("secondaryRelThCi95", None)
 
         # check if DEM in Inputs has desired mesh size
-        pathToDem = dP.checkDEM(cfgSim, inputSimFiles["demFile"])
+        pathToDem = dP.checkRasterMeshSize(cfgSim, inputSimFiles["demFile"], "DEM")
         cfgSim["INPUT"]["DEM"] = pathToDem
 
+        # check if RELTH in Inputs has desired mesh size
```

Comment:
maybe add if not search in Inputs/remeshedRasters if not found remesh

Reply:
I don't think it adds much here, since the function description has it in. It is also not mentioned in the line above for the DEM...